client/my-sites/upgrades/domain-management/controller.jsx

```diff
@@ -28,9 +29,9 @@ import titleActions from 'lib/screen-title/actions';
 const productsList = new ProductsList(),
 	sites = new SitesList();
 
-const renderPage = function( component ) {
+const renderPage = function( component, context ) {
 	ReactDom.render(
-		component,
+		React.createElement( Provider, { store: context.store }, component ),
```

Comment:
@dmsnell: 
I added `context` as a second argument in `renderPage`: https://github.com/Automattic/wp-calypso/pull/1916/files#diff-25e977975fccbfa47a68e14e3ea4e45fR32
I modified this function to wrap components it gets in redux `Provider`.
Provider, to work properly, needs reference to redux store, which, in turn is stored in context and put there in `boot/index.js`.
We only get this context from `page` when it's handling a route. In this file, these route handlers are defined and they receive context as an argument.
So I just passed it down to reach render.